From 9ad2ae4fa05b475b9bd27c879565530f5d0f8fe5 Mon Sep 17 00:00:00 2001
From: Karan Patidar <karan.patidar@intel.com>
Date: Mon, 29 Apr 2019 17:18:46 +0530
Subject: [PATCH] Enable Master Playback Swtich

Updating default value of master playback switch to 1
for CLK target.

Tracked-On: OAM-80076
Signed-off-by: Karan Patidar <karan.patidar@intel.com>
---
 common/audio/default/mixer_paths_0.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/audio/default/mixer_paths_0.xml b/common/audio/default/mixer_paths_0.xml
index 1c114b2..1f8f905 100644
--- a/common/audio/default/mixer_paths_0.xml
+++ b/common/audio/default/mixer_paths_0.xml
@@ -1,7 +1,7 @@
 <mixer>
   <!-- These are the initial mixer settings -->
   <ctl name="Master Playback Volume" value="87" />
-  <ctl name="Master Playback Switch" value="0" />
+  <ctl name="Master Playback Switch" value="1" />
   <ctl name="Headphone Playback Switch" value="1 1" />
   <ctl name="Speaker Playback Switch" value="0 0" />
   <ctl name="Capture Switch" value="0 0" />
-- 
2.20.1

